CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 * Add `DB::cancel_all_background_work` method (stanislav-tkach)
+* Bump `librocksdb-sys` up to 6.13.3 (aleksuss)
```

Comment:
I'm confused a bit, this bump to 6.11.4 was done in #452 (and is published)
what this PR did is fixed clippy warnings

Reply:
Yeah. You're right. There was some troubles with compilation rocksdb 6.13.3 and therefore I accidentally decreased a version to 6.11.4. Thanks for the notify. I'll bump librocksdb-sys version in near future.

Reply:
Thanks for the clarification, so you've left it out for "future work" :)

Reply:
Exactly :)